src/odinapi/static/js/level1calendar.js

```diff
@@ -1,6 +1,14 @@
 // Functions used to populate calendar view:
 
-function getStartView(date) {
+
+require( 'fullcalendar' );
```

Comment:
why the use of require instead of imports?

Reply:
I think `import * as fullcalendar from 'fullcalendar'` is equal without the need of `require`.

Reply:
I used require because that is how it was done in all webpack guides.

Reply:
require for the webpack config file, yes, but why would you have require in your own code if there's no need?

Reply:
I have replaced require with import.